compiler/nimblecmd.nim

```diff
@@ -129,6 +129,10 @@ proc addPathRec(conf: ConfigRef; dir: string, info: TLineInfo) =
 proc nimblePath*(conf: ConfigRef; path: AbsoluteDir, info: TLineInfo) =
   addPathRec(conf, path.string, info)
   addNimblePath(conf, path.string, info)
+  let i = conf.nimblePaths.find(path)
+  if i != -1:
+    conf.nimblePaths.delete(i)
+  conf.nimblePaths.add(path)
```

Comment:
You're moving matching paths to the end of the sequence; is that what we want for cumulative configuration?

I would expect the opposite, if it matters at all, to be consistent with `searchPaths` semantics.

Reply:
I originally had it in reverse but then calling addPath also inserts to the front so only one needs to be in the reverse order.

Reply:
But it's not about the end result, it's about the fact that we want to encode the knowledge of priority in evaluation in a consistent manner so that it can be consistently interpreted by tools.

Reply:
Switched this around to be consistent with searchPaths and lazyPaths.